heartbeat/aws-vpc-move-ip

```diff
@@ -300,7 +320,11 @@ ec2ip_get_and_configure() {
 		return $OCF_ERR_GENERIC
 	fi
 	ocf_log debug "network interface id associated MAC address ${MAC_ADDR}: ${EC2_NETWORK_INTERFACE_ID}"
+	echo $EC2_NETWORK_INTERFACE_ID
+}
 
+ec2ip_get_and_configure() {
+  EC2_NETWORK_INTERFACE_ID="$(ec2ip_get_instance_eni)"
```

Comment:
This also needs to be updated to follow the same indentation as the rest of the agent.